src/work-allocation-2/containers/task-assignment/task-assignment-container.component.spec.ts

```diff
@@ -98,7 +98,9 @@ describe('TaskAssignmentContainerComponent', () => {
 
     wrapper.tasks = null;
     window.history.pushState({ returnUrl: 'mywork/list', showAssigneeColumn: false }, '', 'mywork/list');
```

Comment:
would this be my-work/list?

Reply:
@rayliangatos - interestingly, the tests don't seem to depend on this but, for the sake of consistency, I'll change it to `'my-work/list'`.